openfisca_core/entities/tests/test_group_entity.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any
+
+import pytest
+
+from openfisca_core import entities
+
+from ..typing import HasKey
+
+
+@pytest.fixture
+def parent() -> str:
+    """A key."""
+
+    return "parent"
+
+
+@pytest.fixture
+def uncle() -> str:
+    """Another key."""
+
+    return "uncle"
+
+
+@pytest.fixture
+def first_parent() -> str:
+    """A sub-role."""
+
+    return "first_parent"
+
+
+@pytest.fixture
+def second_parent() -> str:
+    """Another sub-role."""
+
+    return "second_parent"
+
+
+@pytest.fixture
+def third_parent() -> str:
+    """Yet another sub-role."""
+
+    return "third_parent"
+
+
+@pytest.fixture
+def role(parent: str, first_parent: str, third_parent: str) -> Any:
+    """A role."""
+
+    return {"key": parent, "subroles": [first_parent, third_parent]}
+
+
+@pytest.fixture
+def group_entity(role: HasKey) -> entities.GroupEntity:
+    """A group entity."""
+
+    return entities.GroupEntity("key", "label", "plural", "doc", [role])
+
+
+def test_init_when_doc_indented() -> None:
+    """De-indent the ``doc`` attribute if it is passed at initialisation."""
+
+    key = "\tkey"
+    doc = "\tdoc"
+    group_entity = entities.GroupEntity(key, "label", "plural", doc, [])
+    assert group_entity.key == key
+    assert group_entity.doc != doc
```

Comment:
This does not test that there is de-indentation, but that the strings are different. This approximation is too broad and can give false negatvies too easily. What about:

```suggestion
    assert group_entity.doc == doc.lstrip()
```